### DISCUSSION TITLE
PLease🙏 Need help with WebView2 🦖

### DISCUSSION THREAD

**Discussion by GCX-Monter** [unoplatform/uno#17360] [Q&A]
I've been working with your truly wonderful UNO platform for about half a year now. I really like it; it's a real breakthrough! However, I've encountered a problem that I haven't been able to solve for a long time, and because of this, I can't release the application.

Could you please tell me how to set up **WebView2** on Android so that an external link opens in the default browser? For example, if the initial opening link was application-page.com, and then I wanted to go to some-external-link.com from there, this website would open in the browser. There is a similar function in the regular WebView, known as 'ShouldOverLoading, but I can't find anything like that here. Please provide an example of how to implement this correctly. I would be very very grateful ❤️❤️❤️🙏

P.S. If such a possibility doesn't exist at the moment, can you suggest any alternative solution? The main thing is to prevent links from continuing to open in the main window.

**Comment by MartinZikmund** [comment]
@MonteGraph sorry to hear that, can you please create an issue for this? Are you able to create a repro that works in WinUI and does not work on Android? We will investigate.

**Reply by GCX-Monter** [reply]
Hello Martin! Thank you for your reply, I'm very pleased that you answered me. I watched several of your videos when I was just getting acquainted with UNO. So, I'll try to do what you said.
